CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# INDIGO IAM developer guide
+
+Thank you for investing your time in contributing to our project!
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## Development environment
+
+The INDIGO IAM service is a [Maven][maven] project build with Java 17.  
+Run
+
+```
+$ mvn package
+```
+
+to build the project, or
+
+```
+$ mvn package -DskipTests
+```
+to skip tests execution.
+
+You can use your favorite IDE for development (Eclipse is the one adopted at the time of writing).
+Install the `Spring Tools 4` plugin to use Spring buttons and configurations. Please use a modified Google style
+formatter available [here](utils/codestyle-formatter-CNAF.xml) to format your code.
+
+### Run the app
+
+The main package is __iam-login-service__, listening by default on http://localhost:8080. To run it
+
+- enable `h2-test`, `dev` profiles: these profiles allow to run the app in developer mode, where an in memory database is enabled and populated with test users, clients, groups, etc. A web interface of the database is available at http://localhost:8080/h2-console
```

Comment:
Spring profiles

Reply:
"dev" should be enough

Reply:
no, also h2 is necessary

Reply:
Maybe add the default username, password and jdbc connection string for the h2 db?